### DISCUSSION TITLE
更新后小图标无法使用，更新后变更小图标还是详细模式

### DISCUSSION THREAD

**Discussion by molichen88** [hslr-s/sun-panel#210] [Q&A] [answered]
更新后小图标显示无法使用，更新后变更小图标还是详细模式

**Comment by hslr-s** [comment, marked as answer]
该设置已经独立到分组，具体参考
![image](https://github.com/user-attachments/assets/8ed9dacb-25a0-4b3a-8a7c-83969b7517f7)

**Reply by molichen88** [reply]
找到了，谢谢。
